### ISSUE TITLE
Make special interval displays consistent with IntervalSets

### ISSUE THREAD

**Issue by dlfivefifty** [JuliaApproximation/DomainSets.jl#27]
I'm thinking instead of `the chebyshev interval [-1,1]`, maybe `-1 .. 1 (Chebyshev)`

**Comment by daanhb** [comment]
It seems this was added somewhere along the way:
```
julia> ChebyshevInterval()
-1.0..1.0 (Chebyshev)
```

While we're at it, we could have:
```
julia> DomainSets.EmptySpace()
{} (empty domain)
```
and
```
julia> DomainSets.FullSpace()
{x} (complete space)
```
and I would propose to rename `EmptySpace` to `EmptyDomain` and to export it. I'm not sure about `FullSpace`. `WholeDomain` ? `Complete domain`?

**Comment by daanhb** [comment]
This is now in pull request #50. I've started using EmptySpace (now EmptyDomain) quite extensively elsewhere.

**Comment by daanhb** [comment]
I see that ApproxFun already uses `EmptyDomain` for something else, so I'll revert back to `EmptySpace` like it was. We can always change it later.

**Comment by dlfivefifty** [comment]
Don’t worry about ApproxFun, probably it’s just the same thing

**Comment by daanhb** [comment]
I've kept EmptySpace for now, since the other one is still called FullSpace. Closed by #50.
